GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Able/RO_Able_Engines.cfg

```diff
@@ -0,0 +1,33 @@
+//	================================================================================
+//	Able Configs
+//	================================================================================
+
+
+// AJ10-37
+@PART[bluedog_Able_Engine]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[engineSwitch]] {}
+	%engineType = AJ10_Early
+}
+@PART[bluedog_Able_Engine]:AFTER[RealismOverhaulEngines]
+{
+	@title = AJ10-37

```

Comment:
There's no reason to rename this if you aren't gutting every config but the AJ10-37